### DISCUSSION TITLE
Frotz backend: Documentation missing?

### DISCUSSION THREAD

**Discussion by citronalco** [SpectrumIM/spectrum2#418] [Q&A]
Are there any configuration options?
I also have no clue where to put the games.

**Comment by vitalyster** [comment]
From the current directory of the backend executable :) https://github.com/SpectrumIM/spectrum2/blob/master/backends/frotz/main.cpp#L234

**Comment by citronalco** [comment]
Copied zork1.dat to /usr/bin, to /root, to / (spectrum2 runs as root), tried again with zork1.dat renamed to zork1.z5, always restarting spectrum2.
Regardless what I do: When I type "help", the "List of games" is empty.
zork1.dat is ok, works fine with sdlfrotz.

Any further recommendations?

**Reply by vitalyster** [reply]
Use .z5 extension and /var/lib/spectrum2 directory (it should be the current directory for spectrum processes)

**Comment by citronalco** [comment]
Nearly:
/var/lib/spectrum/COMPONENT-JID/zork1.v5 works
Thank you!

**Comment by vitalyster** [comment]
z5 not v5?

**Reply by citronalco** [reply]
Ups, typo.
/var/lib/spectrum/COMPONENT-JID/zork1.z5 works

**Reply by vitalyster** [reply]
/var/lib/spectrum2 I think :)

**Reply by citronalco** [reply]
You're right, again. Embarrassing...

So for anyone sent here by search engines:
/var/lib/spectrum2/COMPONENT-JID/zork1.z5
